playwright/e2e/ui/pages/home.py

```diff
@@ -142,7 +134,7 @@ def highlights_option_page_is_visible(self):
 
     @property
     def highlights_option_page_menu_is_visible(self):
-        return self.page.get_by_test_id("highlight-dropdown-menu-toggle").get_by_role("button").first
+        return self.page.get_by_test_id("highlight-dropdown-menu-toggle").first
```

Comment:
Ah I missed the get_by_role before, sorry about that, I could have saved you some time if I'd noticed. This looks good though.